docs/docs/06-tools/01-cli.md

```diff
@@ -139,10 +152,10 @@ Usage:
 
 ```sh
 $ export CDK_STACK_NAME="my-project"
-$ wing compile --target awscdk app.w
+$ wing compile [entrypoint] --target awscdk
```

Comment:
Why reorder?

Reply:
It looked better to me to follow the same order everywhere with the `[entrypoint]` soon after the command (`wing run|it` doesn't have an option for example). If you think that it was better the other way around, I can revert this change.

Reply:
I don't feel strongly about it